src/main/java/Cars.java

```diff
@@ -0,0 +1,30 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cars {
+    private final List<Car> cars;
+
+    public Cars(List<Car> cars) {
+        this.cars = new ArrayList<>(cars);
+    }
+
+    public void moveAll() {
+        for (Car car : cars) {
+            int number = (int) (Math.random() * 10);
+            car.move(number);
+        }
+    }
+
+    public List<Car> getWinners() {
+        int max = getMaxPosition();
+        return cars.stream().filter(car -> car.getPosition() == max).toList();
```

Comment:
```suggestion
return cars.stream()
           .filter(car -> car.getPosition() == max)
           .collect(Collectors.toList());
```

stream의 경우엔 한줄에 모든걸 표현하면 가독성이 좋지 않아요.
각 stream 메서드마다 이렇게 줄바꿈을 하면 어떨까요?